esileapclient/osc/v1/node.py

```diff
@@ -48,10 +46,10 @@ def take_action(self, parsed_args):
 
         data = client.node.list(filters)
 
-        # These two are the same right now, they just give the name
         if parsed_args.long:
             columns = NODE_RESOURCE.detailed_fields.keys()
             labels = NODE_RESOURCE.detailed_fields.values()
+            print(columns)
```

Comment:
This  probably isn't needed?

Reply:
I always forget to remove my debugging prints oops